src/accordian/Accordion.tsx

```diff
@@ -114,12 +117,11 @@ export class Accordion extends React.Component<AccordionProps, AccordionState> {
         );
     }
 
-    getItemContent = (index: any, title: any, isPrevious: boolean) => {
+    //get content for each accordion panel
+    getItemContent = (index: any, title: any, isOpen: boolean | undefined) => {
         const {accordionMultiPanel} = this.props;
-        const panelClass = isPrevious
-            ? ClassNames.ACCORDION_PANEL_INNER
-            : classNames([ClassNames.ACCORDION_PANEL_INNER, ClassNames.ACCORDION_PANEL_OPEN]);
-        const expanded = !isPrevious;
+        const panelClass = classNames([ClassNames.ACCORDION_PANEL_INNER, isOpen && ClassNames.ACCORDION_PANEL_OPEN]);
+        const expanded = isOpen;
```

Comment:
can't we use isOpen directly at line 133 ?

Reply:
Sure. I'll add that change while adding version number.